devops/scripts/combine-junit-test-results.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -u
+set -e
+
+
+# Directory should already exist, from the CI Molecule scenario,
+# but let's be certain.
+mkdir -p junit
+
+# Combine tests for export as artifacts. There are at least two XML files,
+# one from the application tests, and another from the config tests.
+./molecule/testinfra/combine-junit.py ./*results.xml > ./junit/junit.xml
```

Comment:
I'm not sure about moving all the tests into a shared molecule/testinfra folder.. its not really a scenario (like the other folders) so this could be confusing later on. :man_shrugging: i guess its okay. just talking out loud here

Reply:
@msheiny Totally agree, perhaps `devops/scripts/` would be the best location?

Reply:
> `devops/scripts/`

yeah i could get down with that :)